ClosedXML/Excel/Sparkline/XLSparkLineGroup.cs

```diff
@@ -0,0 +1,263 @@
+        }
+
+        public Boolean RightToLeft { get; set; }
+        public IXLSparklineGroup SetRightToLeft(Boolean rightToLeft)
+        {
+            RightToLeft = rightToLeft;
+            return this;
+        }           
+
+        public Double ManualMin { get; set; }
+        public IXLSparklineGroup SetManualMin(Double manualMin)
+        {
+            ManualMin = manualMin;
+            return this;
+        }
+
+        public Double ManualMax { get; set; }
+        public IXLSparklineGroup SetManualMax(Double manualMax)
+        {
+            ManualMax = manualMax;
+            return this;
+        }
+
+        public Double LineWeight { get; set; }
+        public IXLSparklineGroup SetLineWeight(Double lineWeight)
+        {
+            LineWeight = lineWeight;
+            return this;
+        }
+
+        public XLSparklineType Type { get; set; }
+        public IXLSparklineGroup SetType(XLSparklineType type)
+        {
+            Type = type;
+            return this;
+        }
+
+        public XLDisplayBlanksAsValues DisplayEmptyCellsAs { get; set; }
+        public IXLSparklineGroup SetDisplayEmptyCellsAs(XLDisplayBlanksAsValues displayEmptyCellsAs)
+        {
+            DisplayEmptyCellsAs = displayEmptyCellsAs;
+            return this;
+        }
+
+        public XLSparklineAxisMinMax MinAxisType { get; set; }
+        public IXLSparklineGroup SetMinAxisType(XLSparklineAxisMinMax minAxisType)
+        {
+            MinAxisType = minAxisType;
+            return this;
+        }
+
+        public XLSparklineAxisMinMax MaxAxisType { get; set; }
+        public IXLSparklineGroup SetMaxAxisType(XLSparklineAxisMinMax maxAxisType)
+        {
+            MaxAxisType = maxAxisType;
+            return this;
+        }
+
+        public IXLWorksheet Worksheet { get; }
+
+        public void CopyTo(IXLWorksheet targetSheet)
+        {
+            var newSlg = new XLSparklineGroup(this, targetSheet);
+            targetSheet.SparklineGroups.Add(newSlg);
+
+            foreach (var sl in this)
+            {
+                newSlg.AddSparkline(targetSheet.Cell(sl.Cell.Address), targetSheet.Range(sl.Formula.Value).RangeAddress.ToStringRelative(true));
+            }
+        }
+
+        public void CopyFrom(IXLSparklineGroup sparklineGroup)
+        {
+            AxisColor = XLColor.FromColor(sparklineGroup.AxisColor.Color);
+            SeriesColor = XLColor.FromColor(sparklineGroup.SeriesColor.Color);
+            MarkersColor = XLColor.FromColor(sparklineGroup.MarkersColor.Color);
+            HighMarkerColor = XLColor.FromColor(sparklineGroup.HighMarkerColor.Color);
+            LowMarkerColor = XLColor.FromColor(sparklineGroup.LowMarkerColor.Color);
+            FirstMarkerColor = XLColor.FromColor(sparklineGroup.FirstMarkerColor.Color);
+            LastMarkerColor = XLColor.FromColor(sparklineGroup.LastMarkerColor.Color);
+            NegativeColor = XLColor.FromColor(sparklineGroup.NegativeColor.Color);
```

Comment:
I think this is in the CopyFrom method? If I assign the NegativeColor to the NegativeColor would it not be the same XLColor object as the object we are copying from? It's meant to be different so that you can change one without changing the other.

Reply:
XLColor cannot be changed by it's own. Moreover, each occurence of the same color (say, black) is actually the same instance. By direct assigning you save time by not searching the color in the repository by its key.
And when I asked about NegativeColor I meant all the color properties in this section, of course.